notebooks/OpenScale.ipynb

```diff
@@ -136,15 +136,15 @@
    "outputs": [],
    "source": [
     "WML_CREDENTIALS = {\n",
-    "  \"apikey\": \"kjfkdjfkdjfkdjkdjfkdjfkdfj\",\n",
-    "  \"iam_apikey_description\": \"Auto generated apikey during resource-key operation for Instance - crn:v1:bluemix:public:pm-20:us-south:a/fkjkfdkfjkdjkdjfkdjfkdfjdkfabb::\",\n",
-    "  \"iam_apikey_name\": \"auto-generated-apikey-kdjfkjdkdjf99938493893e3\",\n",
+    "  \"apikey\": \"***,\n",
+    "  \"iam_apikey_description\": \"Auto generated apikey during resource-key operation for Instance - crn:v1:bluemix:public:pm-20:us-south:a/47b84451ab70b94737518f7640a9ee42:4a7f62cc-f564-4144-8af4-b9f53533d2cf::\",\n",
```

Comment:
not sure what the guid is.  OK or wanna go back to fkjkfdkf or ****?

Reply:
Fixed